packages/devextreme/js/__internal/viz/chart_components/m_advanced_chart.ts

```diff
@@ -10,6 +10,8 @@ import {
 } from '@js/viz/core/utils';
 import rangeDataCalculator from '@js/viz/series/helpers/range_data_calculator';
 import { Range } from '@js/viz/translators/range';
+// @ts-expect-error
```

Comment:
Which particular error? And why?

Reply:
no file error. i believe this is happening because js file is imported into ts file,  i used the same approach with @ts-expect-error as we use in other places in our project, so this is not something new